packages/docs/src/examples/grid/AgGrid.vue

```diff
@@ -62,5 +60,6 @@ export default {
 </script>
 
 <style lang="scss">
-@import "ag-grid-community/dist/styles/ag-grid.css";
+@import "../../../../../node_modules/ag-grid-community/dist/styles/ag-grid.css";
```

Comment:
I know we are not able to show the style tag for now. But I think this is important to be shown in this demo. It would be nice to use Webpack aliases there instead of ugly relative paths. We can do it later, but as far as I know, we want to make it so that users can copy this demo code, paste it into their own project and it should work fine.

```suggestion
@import "ag-grid-community/dist/styles/ag-grid.css";
@import "vuestic-ui/styles/ag-theme-vuestic/ag-theme-vuestic.scss";
```

Reply:
I think it is not important for now. You can skip it.